ClassificationNeuralNetwork.java

```diff
@@ -62,4 +82,41 @@ public static void readDataFromFile()
 					+ currLineNum);
 		}
 	}
+	
+	public static void putDataInArrayList()
+	{
+		for(int i=0; i<numLines; i++)
+		{
+			double[] toAdd= new double[10];
+			
+			toAdd[0]=radii[i];
+			toAdd[1]=textures[i];
+			toAdd[2]=perimeters[i];
+			toAdd[3]=area[i];
+			toAdd[4]=smoothness[i];
+			toAdd[5]=compactness[i];
+			toAdd[6]=concavity[i];
+			toAdd[7]=concavePnts[i];
+			toAdd[8]=symmetry[i];
+			toAdd[9]=fractalDim[i];
+			
+			listOfTestData.add(toAdd);
+			listOfTestClassifications.add(classification[i]);
+		}
+	}
+	
+	public static void shuffle()
+	{
+		int eightyPercentOfData= (int) (listOfTestData.size()*.8);
+		
+		for(int i=0; i<eightyPercentOfData; i++)
+		{
```

Comment:
This method may not be fully utilizing the arraylist.

Reply:
What do you mean?

Reply:
So when we originally spoke, we mentioned using the arraylist.shuffle function to make the shuffling easier. But the code you have is logically sound and, from what I could tell in my testing, shuffles things just fine. This was more of just pointing out that there may have been a simpler way to implement this. There's no need to change anything though.

Reply:
I thought this made more sense than a shuffle method. That's why it's not implemented.